PURL-TYPES.rst

```diff
@@ -221,6 +207,34 @@ nuget
 
       pkg:nuget/EnterpriseLibrary.Common@6.0.1304
 
+oci-artifact
+------------
+``oci-artifact`` for all artifacts stored in OCI Distribution-spec conformant
```

Comment:
Can we link to the OCI Distribution-spec here?

Reply:
How about this, @iamwillbar?

```suggestion
``oci-artifact`` for all artifacts stored in registries that conform to the `OCI
Distribution Specification <https://github.com/opencontainers/distribution-spec>`_,
including container images built by Docker and others:
```